handout/http/catching_rejections.md

```diff
@@ -0,0 +1,91 @@
+## Catching Rejections
+
+To catch rejections we use the subscriber's `error` and `complete` callbacks.
+
+```ts
+authenticate(username, password) {
```

Comment:
For some reason I find it harder to understand the snippet without knowing the name of the class of if you are referring to a component or a service. Would you mind adding the class name to the snippet?